README.md

```diff
@@ -37,6 +37,17 @@ A config template has been placed in `/etc/hubble/hubble`. Modify it to your spe
 
 The first two commands you should run to make sure things are set up correctly are `hubble --version` and `hubble test.ping`.
 
+### Running in Docker container
+You can quickly get up and running with any supported distribution simply by installing net-tools in the running the docker container.
+To pull and run an image `docker run -it {distroy:tag} sh`, for example:
+```
+docker run -it centos:7 sh
+```
+Install net-tools
+```yum install net-tools
```

Comment:
Can you put the triple backticks on their own line? The flavor of markdown that github uses allows you to specify a syntax highlighting language on the same line as your triple backticks, and that feature is consuming your yum command.